scripts/mkcloud

```diff
@@ -588,10 +588,10 @@ function createcloudsnapshot()
     for i in $allnodeids ; do
         wait_for_node_shutdown $cloud-node$i
     done
-    for d in $(lvs | grep $cloudvg | grep snap | awk '{print $1}') ; do
+    for d in $(lvs ${cloudvg} | grep ${cloud}\. | grep snap | awk '{print $1}') ; do
```

Comment:
I think, it would be nicer to combine the two greps into one - e.g.
```bash
grep "${cloud}\..*\.snap"
```

and same below in Line 607